libs/go/sia/aws/agent/agent.go

```diff
@@ -567,6 +579,32 @@ func RunAgent(siaCmd, siaDir, ztsUrl string, opts *options.Options) {
 			stop <- true
 		}()
 
+		go func() {
```

Comment:
how do we guarantee that when the goroutine is processed, there are valid identity certs already fetched otherwise our token fetch operation will not succeed?

Reply:
done, thx